pallets/phala/src/compute/basepool.rs

```diff
@@ -200,6 +227,7 @@ pub mod pallet {
 			Pallet::<T>::set_nft_attr(self.cid, self.nftid, &self.attr)?;
 			Ok(())
 		}
+		/// Deconstruct nftguard proactively. Used in the read-only case.
```

Comment:
```suggestion
		/// Deconstructs the [`NftGuard`] proactively. Used in the read-only case.
```

Reply:
Please note the third-person singular (i.e. descriptive) in the first line of the function comment

Reply:
done

Reply:
Not done

Reply:
done